components/stripe/stripe.app.js

```diff
@@ -2,15 +2,19 @@
 
 const stripe = require("stripe");
 
-const createOptionsMethod = (collectionOrFn, keysOrFn) => async function ({ prevContext }) {
+const createOptionsMethod = (collectionOrFn, keysOrFn) => async function ({
```

Comment:
I would recommend `/** jsdoc */` this function since it abstracts a few things in it IMO. But I totally understand if you don't document it since it will be out of scope I guess.